core/Plugin/Report.php

```diff
@@ -849,15 +849,16 @@ public function getDimensions()
             $dimensions[$dimensionId] = $this->getDimension()->getName();
         }
 
-        if (!empty($this->getSubtableDimension())) {
-            $subDimensionId = str_replace('.', '_', $this->getSubtableDimension()->getId());
-            $dimensions[$subDimensionId] = $this->getSubtableDimension()->getName();
-        }
-
-        if (!empty($this->getThirdLeveltableDimension())) {
-            $subDimensionId = str_replace('.', '_', $this->getThirdLeveltableDimension()->getId());
-            $dimensions[$subDimensionId] = $this->getThirdLeveltableDimension()->getName();
-        }
+        $level = 1;
```

Comment:
@sgiehl This seems to be going in a loop for customDimensions

![image](https://github.com/user-attachments/assets/7cca201b-3e1a-4673-a762-322a6534ddfa)

Reply:
Strange. For me that worked without problems. I've now limited the loop to 100. Guess more than 100 dimensions should be unlikely. Let me know if you still have problems.

Reply:
@sgiehl This works :+1: